File: PHP/7php 1/generate_integers_mn.php
<?php
/*
|--------------------------------------------------------------------------
| Series of integers from m to n
|--------------------------------------------------------------------------
|
*/

function generate_integers(int $m, int $n): array {
   $res = [];
   for ($i=$m; $i <= $n; $i++) { 
      array_push($res, $i);
   }
   return $res;
}


function generate_integers(int $m, int $n): array {
   return range($m, $n);
}


function generate_integers(int $start, int $stop): array {
   $res = [];
   while($start != $stop + 1) {
      $res[] = $start;
      $start++;
   }
   return $res;
}


function generate_integers(int $m, int $n): array {
   $range = range($m, $n);
   foreach($range as $numbers){
      return $range;
   }
}


print_r(generate_integers(2, 5));

File: Python/7py 2/generate_integers_mn.py
"""
|--------------------------------------------------------------------------
| Series of integers from m to n
|--------------------------------------------------------------------------
|
"""

def generate_integers(m, n): 
   res = []
   for i in range(m, n+1):
      res.append(i)
   return res


def generate_integers(m, n): 
   return list(range(m,n+1))


def generate_integers(m, n):
   return [i for i in range(m,n+1)]


def generate_integers(m, n): 
   return [*range(m, n + 1)]


print(generate_integers(2, 5))